packages/core/data-transfer/src/file/providers/destination/index.ts

```diff
@@ -61,10 +63,22 @@ class LocalFileDestinationProvider implements IDestinationProvider {
 
   #archive: { stream?: tar.Pack; pipeline?: Stream } = {};
 
+  #diagnostics?: IDiagnosticReporter;
+
   constructor(options: ILocalFileDestinationProviderOptions) {
     this.options = options;
   }
 
+  #reportInfo(message: string) {
+    this.#diagnostics?.report({
+      details: {
+        createdAt: new Date(),
+        message: `${chalk.cyan(`[file-destination-provider]`)} ${message}`,
```

Comment:
Instead of embedding the source into the messages, could we add a `source` identifier and have the logger add it in? That way we don't have to manage it in every provider, we can add filters for it, etc